components/membership/registration-impl/src/test/kotlin/net/corda/membership/impl/registration/dynamic/handler/mgm/ApproveRegistrationHandlerTest.kt

```diff
@@ -326,12 +187,46 @@ class ApproveRegistrationHandlerTest {
     fun `invoke updates the MGM's view of group parameters with notary, if approved member has notary role set`() {
         val state = RegistrationState(registrationId, notary.toAvro(), owner.toAvro())
 
-        handler.invoke(state, key, command)
+        val results = handler.invoke(state, key, command)
 
-        verify(membershipPersistenceClient).addNotaryToGroupParameters(
+        verify(membershipPersistenceClient, times(1)).addNotaryToGroupParameters(
```

Comment:
```suggestion
        verify(membershipPersistenceClient).addNotaryToGroupParameters(
```
Default is checking for a single call.